test/libsolidity/SemanticTest.cpp

```diff
@@ -142,6 +147,31 @@ void SemanticTest::initializeBuiltins()
 		soltestAssert(_call.arguments.parameters.empty(), "No arguments expected.");
 		return toBigEndian(u256(storageEmpty(m_contractAddress) ? 1 : 0));
 	};
+	m_hooks.emplace_back(
+		[](FunctionCall const& _call) -> std::vector<std::string>
```

Comment:
```suggestion
		[](FunctionCall const& _call) -> vector<string>
```
There are also a few more above and below.